src/test/java/co/rsk/federate/FederatorSupportTest.java

```diff
@@ -140,7 +141,15 @@ public void getPegoutCreationRskTxHashByBtcTxHash_zero_hash() {
         co.rsk.bitcoinj.core.Sha256Hash sha256Hash = co.rsk.bitcoinj.core.Sha256Hash.ZERO_HASH;
         Keccak256 hash = Keccak256.ZERO_HASH;
 
-        test_getPegoutCreationRskTxHashByBtcTxHash(sha256Hash, hash, true);
+        test_getPegoutCreationRskTxHashByBtcTxHash(sha256Hash, hash, false);
+    }
+
+    @Test
+    public void getPegoutCreationRskTxHashByBtcTxHash_keccak256_from_a_zero_byte_array() {
+        co.rsk.bitcoinj.core.Sha256Hash sha256Hash = co.rsk.bitcoinj.core.Sha256Hash.ZERO_HASH;
+        Keccak256 hash = new Keccak256(new byte[32]);
+
+        test_getPegoutCreationRskTxHashByBtcTxHash(sha256Hash, hash, false);
```

Comment:
If exists == false, FederatorSupport mock will always return `Keccak256.ZERO_HASH.getBytes()`. So this test is pretty much the same as the one above, it doesn't really matter what value you use for the rsk hash

Reply:
Ohhh, you are right. I'll try to change that. Thanks!